gtsam/linear/VectorValues.h

```diff
@@ -263,6 +263,11 @@ namespace gtsam {
     /** equals required by Testable for unit testing */
     bool equals(const VectorValues& x, double tol = 1e-9) const;
 
+    /// Check exact equality.
+    friend bool operator==(const VectorValues& lhs, const VectorValues& rhs) {
+      return lhs.equals(rhs);
```

Comment:
But this is not exact….

Reply:
Still uses equals, no?

Reply:
I edited the docstring and it says `/// Check equality.` Is that not showing for you?

Reply:
operator== implies strict equality. If you want equals, then please rename to `equals` and provide a tolerance parameter.